core/src/main/java/org/apache/iceberg/hadoop/HadoopTables.java

```diff
@@ -70,18 +72,23 @@ public Table load(String location) {
    *
    * @param schema iceberg schema used to create the table
    * @param spec partition specification
+   * @param properties properties of the table to be created
```

Comment:
Can you note that null is accepted?

Also, should we do something similar for spec? If spec is null, we could use `PartitionSpec.unpartitioned()`.

Reply:
Sure, please see the latest code change.